modules/rostests/apitests/user32/SystemParametersInfo.c

```diff
@@ -106,11 +106,130 @@ MSG_ENTRY CaptionHeight_chain[]={
 static void Test_NonClientMetrics()
 {
     NONCLIENTMETRICS NonClientMetrics;
```

Comment:
That one has a problem, namely it's not explicitly told whether it's the unicode or ANSI version here (it should be unicode because we compile everything in unicode, but it makes the test look as if it was written with preconceptions).